### DISCUSSION TITLE
Read file instead of folder

### DISCUSSION THREAD

**Discussion by wackoracoon** [MiczFlor/RPi-Jukebox-RFID#1737] [Q&A]
Hello,

I’m a long time user of one of the ancestor of phoniebox : music-cards.
It’s running like a charm since 5 years for my daughter but I’m thinking of upgrading the solution and install phoniebox. I tested it on a « preprod » raspberry but I’ll a little bit confused about the new philosophy : instead of playing one file for a card, one card plays a folder.
It’s a problem for me because I have about 150 cards linked to files well organized in folder/subfolder and severals files in folders etc.

Can you confirm me that linking a card to a file is impossible ?

I haven't got the motivation to rebuild everything yet :/

**Comment by pabera** [comment]
Hey!

>  instead of playing one file for a card, one card plays a folder.

I am not sure what version you use but the following statement is true for a while (at least for V2.x). You can circumstance it by putting a single file into a folder.

With V3, this is again possible. You choose to play by folder, album or a single song.

> I haven't got the motivation to rebuild everything yet :/

Nevertheless, I am not sure whether your old card database will be compatible with the new versions anyway :-)
But some scripting will do the trick most likely ;-)
